blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/model/BbUser.java

```diff
@@ -11,12 +11,6 @@
  */
 @Restricted(NoExternalUse.class)
 public abstract class BbUser {
-    /**
-     * @return User name
-     */
-    @JsonProperty("name")
-    public abstract String getName();
-
```

Comment:
Is this added back in for non-cloud?

Reply:
Hrm.. i forgot now. I think we found out its not actually used anywhere.